druid/src/contexts.rs

```diff
@@ -271,6 +271,34 @@ impl_context_method!(
         pub fn has_focus(&self) -> bool {
             self.widget_state.has_focus
         }
+
+        /// The disabled status of a widget.
+        ///
+        /// The widget can still be disabled if this value is `false`, use [`is_enabled`] to check if
+        /// the widget is currently enabled.
+        ///
+        /// If the value is `true` this widget will stay disabled at least until [`EventCtx::set_enabled`]
+        /// or [`UpdateCtx::set_enabled`] is called.
+        ///
+        /// [`is_enabled`]: #method.is_enabled
+        /// [`EventCtx::set_enabled`]: struct.EventCtx.html#method.set_enabled
+        /// [`UpdateCtx::set_enabled`]: struct.LifeCycleCtx.html#method.set_enabled
+        pub fn is_set_disabled(&self) -> bool {
```

Comment:
I thought it would be more complete, having the method. Maybe a user wants to access this information, but i also can't think of a good example how to make use of it. Your right, it probably just invites to user to use the `disabled` state in an unsound way. I will remove it.

Reply:
I think it might make sense here to be conservative; we don't include this method initially, and then add it if somebody can demonstrate a need.

Reply:
Ok